xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -299,18 +318,6 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
 
             return layoutFragment
         }
-        
-        // MARK: - NSTextContentStorageDelegate
-        
-        func textContentStorage(_ textContentStorage: NSTextContentStorage, textParagraphWith range: NSRange) -> NSTextParagraph? {
-            guard let attributedSubstring = textContentStorage.textStorage?
```

Comment:
Did the API change in TK2? This form would mean we don't need to bring in the NSRange extension.